phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -96,6 +100,30 @@ public void showAvailableApps() {
      */
     public void populate(List<CategoryDTO> categories) {
         Platform.runLater(() -> {
+            // set default category icons
+            try{
+                String css = "";
+                for (CategoryDTO category : categories) {
+                    css += "." + category.getName().toLowerCase() + "Button{\n";
+                    css += "-fx-background-image: url('" + category.getIcon() + "');\n";
+                    css += "}\n";
+                }
+                File temp = new File("tempfile.css");
+                temp.delete();
+                temp.createNewFile();
+                BufferedWriter bw = new BufferedWriter(new FileWriter(temp));
+                bw.write(css);
+                bw.close();
```

Comment:
I don't really like Writers and BufferedWriters, how about using `Files::write` instead?